client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -205,18 +210,18 @@ class TransferDomainStep extends React.Component {
 	}
 
 	getTransferDomainPrecheck() {
-		const { inboundTransferStatus, submittingWhois } = this.state;
+		const { domain, inboundTransferStatus, submittingWhois, searchQuery } = this.state;
 
 		return (
 			<TransferDomainPrecheck
-				domain={ this.state.domain }
+				domain={ domain || searchQuery }
```

Comment:
Feels weird. We can probably set the `domain` the same way you did `precheck`. :)